test/regressions/index.js

```diff
@@ -121,6 +121,7 @@ const blacklist = [
   'docs-getting-started-templates-sign-in-side/SignInSide.png', // Flaky
   'docs-getting-started-usage/Usage.png', // Not needed
   'docs-guides', // Not needed
+  'docs-landing', // Not needed
```

Comment:
I think it would make sense to run these on some of the components. At least on the ones that don't have external dependencies (e.g. github star button or twitter follow button).

Reply:
I think that we should focus our resources on the components we publish on npm, with an even bigger emphasis on the components that have a lot of dependencies. I would expect a visual test on the landing page to have a low ROI because: we can quickly test the impact of our change, which is not true at all when you change a variable in the theme that can have a broad impact on a dozen of different pages (no time to review them manually)

Reply:
Regarding the error, I believe it comes from a missing context requirement.